.changes/8e87c6dd-5138-4b79-a0c2-255ecf31898b.json

```diff
@@ -0,0 +1,6 @@
+{
+    "id": "8e87c6dd-5138-4b79-a0c2-255ecf31898b",
+    "type": "misc",
+    "description": "Upgrade to latest versions of OkHttp, Okio, Kotlin",
```

Comment:
**Nit**: Prefix with `⚠️ **IMPORTANT**: `. Our **kat** tool does this automatically but if we manually tweak the file we have to manually prefix the description.